scripts/temp_parachain_types.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
@jacogr can you upstream these types please.

Reply:
https://github.com/polkadot-js/api/pull/2895

Reply:
And the apps UI API bump - https://github.com/polkadot-js/apps/pull/4136